tidb-cloud/tidb-cloud-import-local-files.md

```diff
@@ -26,40 +26,45 @@ Currently, this method supports importing one CSV file for one task into either
         >
         > If you have multiple projects, you can switch to the target project in the left navigation pane of the **Clusters** page.
 
-    2. Click the name of your target cluster to go to its overview page, and then click **Import** in the left navigation pane.
+    2. Click **...** at the end of the cluster row and then click **Import**.
```

Comment:
According to PMs, the **Import** option will be removed from ... next week, to avoid updating the doc again, can we tell users to click the name of your target cluster to go to its overview page?